### PR TITLE
intakeUpdate

### PR DESCRIPTION
**Pull request by Rock2307**

### CODE REVIEW

File: src/main/java/frc/robot/GrabberIntake.java [hunk 1]
```diff
@@ -1,21 +1,48 @@
...
+            intakeMotor.setVoltage(Constants.IntakeVoltage);
         } else {
-            grabber.set(Value.kReverse);
+            intakeMotor.setVoltage(-1 * Constants.IntakeVoltage);
+        }
+        nomVolt = intakeMotor.getVoltageCompensationNominalVoltage();
+        SmartDashboard.putNumber("Intake Nominal Voltage", nomVolt);
+        
+        if(nomVolt >= 0) {
+            timer.start();
```
Comment (bgoll1006) [inline src/main/java/frc/robot/GrabberIntake.java:36]: You're starting the timer then checking it right after?

Comment (Rock2307) [reply src/main/java/frc/robot/GrabberIntake.java:36]: Yeah the timer needs a little bit of work. What do you suggest? I pushed to git so there would be some framework available.

Comment (bgoll1006) [reply src/main/java/frc/robot/GrabberIntake.java:47]: I worked on it, should be ok now

**Review by jjstevens5 — APPROVED** [review]

**Review by bgoll1006 — COMMENTED** [review]
looks good

**Pull request merged**